_posts/it/2019-10-28-Call-Developers-Italia-parliamo-di-design-dei-servizi-pubblici.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  Call Developers Italia, parliamo di design dei servizi pubblici
+subtitle: Il team di Designers Italia ospite della Community Call del 30 novembre
+date:   2019-10-28 12:00:00 +0100
+tags: developers call
+categories: news
+author: Andrea Biancini
+image: /assets/icons/logo-it.svg
+locale: it
+ref:
+  en: 
+redirect_from:
+   -
+---
+
+Ciao a tutti!
+
+**mercoledì 30 ottobre alle ore 15** è in programma un nuovo appuntamento con la [Community Call bisettimanale](https://developers.italia.it/it/news/2019/09/03/una-call-per-la-community-di-Developers-Italia) di Developers Italia.
+Dopo aver discusso [nella scorsa call](https://developers.italia.it/it/news/2019/10/11/Appuntamento-in-call-con-Developers-Italia) di **ecosistemi, piattaforme e API**, in questo nuovo appuntamento parleremo di un'altro importante tema per lo sviluppo di servizi pubblici sempre più efficienti: **il design!**
+
+Valorizzando le esperienze e il lavoro fatto nella nostra community, durante la call discuteremo dello stato dell’arte del design nei prodotti e servizi per la Pubblica Amministrazione:
+
+- [Francesco Zaia](https://teamdigitale.governo.it/it/people/francesco-zaia.htm), del Team per la Trasformazione Digitale, ci parlerà dal lavoro e dei risultati raggiunti da parte di [Designers Italia](https://designers.italia.it/);
+- alcuni sviluppatori del Consorzio dei comuni trentini racconteranno la propria esperienza nell'utilizzare [il kit comuni di Designers Italia](https://designers.italia.it/kit/comuni/) per **progettare il sito web di un Comune**. 
```

Comment:
"Alcuni sviluppatori" non è corretto, se stessimo generici con il Consorzio dei comuni trentini racconterà la sua esperienza ... ?

Reply:
ecco @biancini ho provato a riformulare in modo più corretto